webui/tests/tests/StepImplementation.ts

```diff
@@ -211,6 +217,7 @@ export default class StepImplementation {
   @Step('Navigate to experiment list page')
   public async navigateToExperimentList() {
     await goto(`${BASE_URL}/experiments`);
+    await t.waitFor(async () => await t.$(selectors.antdTable).exists());
```

Comment:
this should be simplified

Reply:
I found the same thing in their examples. I'm not sure if it's a bad example or if this necessary. https://docs.taiko.dev/api/waitfor/ 

you'd think `await t.waitFor(t.$(selectors.antdTable));` would be thing

Reply:
might we worth testing it out?

Reply:
there are a couple of places we do this I think I tested it but I can't remember I'll retry.

Reply:
b790a0954988a265ff16dfbd3ea4713803bb2736